README.md

```diff
@@ -570,4 +570,58 @@ const X_CORRELATION_ID = "X-Correlation-ID"
 ```
 
 </td></tr>
-</tbody></table>
\ No newline at end of file
+</tbody></table>
+
+### Hexagonal Architecture
+* Our architecture is separated into `Left`, `Center`, `Right`.
+
+* `Left`
+  * Application code
+  * HTTP and REST for our API
+  * Is free to import center
+  * Must implement interfaces dictated by center
+  * Will drive center
+    * By instantiating aggregates/entities 
+    * By calling methods on center domain object, `-able`'s
+
+* `Center`
+  * Where we define aggregates/entities
+  * Where we define center domain objects, `-able`'s
+  * Where we define our infstracture interfaces, `-or`'s
+  * Center is importable by all
+  * Center drives right
+  * Center imports nothing
+    * Prevents circular imports
+    * Allows repository mocking
+
+* `Right`
```

Comment:
this is only partially correct. there's a finer point about the right side thats missed here. 
* must a "right" always be infrastructure?
* is the right always defined by us? is this different than center or left?